sklearn/preprocessing/_data.py

```diff
@@ -1233,6 +1239,12 @@ def fit(self, X, y=None):
 
             self.scale_ = quantiles[1] - quantiles[0]
             self.scale_ = _handle_zeros_in_scale(self.scale_, copy=False)
+            if self.unit_variance:
+                # Create scipy.stats.norm object
+                output_distribution = getattr(stats, 'norm')
```

Comment:
why this instead of `stats.norm`? Is there any difference?

Reply:
No i don't think so, updated.